src/v2/publish/App.html

```diff
@@ -337,16 +350,14 @@ <h2>{@html __('publish / share-embed') }</h2>
                 setTimeout(() => this.set({ publishing: false }), 500);
                 this.set({ chart: chartInfo });
 
-                if (window.parent) {
-                    window.parent.postMessage(
-                        {
-                            source: 'datawrapper',
-                            type: 'chart-publish',
-                            chartId: chartInfo.id
-                        },
-                        '*'
-                    );
-                }
```

Comment:
I don't see how removing this safety check will improve the situation. If there is no `window.parent` then `window.parent.postMessage` is undefined

Reply:
`window.parent` is always truthy. That makes the check obsolete.

> The Window.parent property is a reference to the parent of the current window or subframe.
>
> If a window does not have a parent, its parent property is a reference to itself.
> https://developer.mozilla.org/en-US/docs/Web/API/Window/parent